src/dataloaders/lra.py

```diff
@@ -93,8 +93,8 @@ def tokens_to_indices(tokens, vocab):
     Returns:
         list: A list of indices corresponding to the input tokens.
     """
-    # TODO: (louis) Could be more elegant and using the unknow token 
-    return [vocab[token] if token in vocab.keys() else -1 for token in tokens]
+
+    return [vocab[token] if token in vocab.keys() else vocab['<unk>'] for token in tokens]
```

Comment:
```suggestion
    return [vocab.get(token, '<unk>') for token in tokens]
```

A likely useless alternative just for fun

Reply:
you forgot to say pythonic and elegant @tianjianjiang 🤝